packages/babel-plugin-proposal-object-rest-spread/test/fixtures/object-rest/assignment-pattern-static/output.js

```diff
@@ -0,0 +1,11 @@
+function foo() {
+  var _props$Global;
+
+  const {
+    Global: {
+      classes
+    } = this
+  } = props,
+        InputPropsOther = babelHelpers.objectWithoutProperties((_props$Global = props.Global, _props$Global !== void 0 ? _props$Global : this), ["classes"]),
```

Comment:
This is accessing `props.Global` twice: once for the destructuring, and once here.

Reply:
Good point! It seems that this issue has been around [for a while](https://github.com/babel/babel/blob/da5e355f995e46601aaeeb67bdae48012c78ea27/packages/babel-plugin-transform-object-rest-spread/test/fixtures/object-rest/nested/actual.js). I would suggest we address it in a separate PR.